src/sql/workbench/contrib/notebook/browser/outputs/plotlyOutput.component.ts

```diff
@@ -51,6 +52,7 @@ export class PlotlyOutputComponent extends AngularDisposable implements IMimeCom
 	private _cellModel: ICellModel;
 	private _bundleOptions: MimeModel.IOptions;
 	private _plotDiv: PlotlyHTMLElement;
+	private _plotly: any;
```

Comment:
We should avoid using any whenever we can. Are you unable to get proper typings for this var?

Reply:
They weren't being used in this file, but I did find the @types/plotly.js module already installed. 

I imported Plotly and also replaced the PlotlyHTMLElement to use the module's version rather than the one defined in the file. @chlafreniere, does this look OK to you?